core/templates/pages/creator-dashboard-page/creator-dashboard-page.component.html

```diff
@@ -218,6 +219,7 @@ <h2 class="oppia-dashboard-title" translate="I18N_DASHBOARD_CREATOR_DASHBOARD"><
           <div class="title-section"
                ng-style="{'background-color': exploration.thumbnailBgColor}">
             <img class="thumbnail-image"
+                 alt="Lightbulb"
```

Comment:
Er.. why lightbulb?

Reply:
Hm, but isn't this dynamic? I think the thumbnail icon varies depending on the category the exploration belongs to. Perhaps we could just call it "Exploration Thumbnail" or be more specific and provide a specific alt text based on the image (it would be one of [these](https://github.com/oppia/oppia/tree/develop/assets/images/subjects)) -- I think you can parse it from the URL).

Reply:
Lightbulb may be the default when no subject / category is specified in the editor.

Reply:
I think these thumbnails might be considered decorative images and empty alts are okay for them. https://www.w3.org/WAI/tutorials/images/decorative/